src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -10,14 +12,21 @@ import { VARIANT_SUCCESS, VARIANT_ERROR, TYPE_LOCAL } from 'containers/Notificat
 import { INCIDENT_URL } from 'signals/incident-management/routes';
 
 import LoadingIndicator from 'components/LoadingIndicator';
-import Form from './Form';
+import IncidentSplitForm from './IncidentSplitForm';
 
 const IncidentSplitContainer = ({ FormComponent }) => {
   const { data, error, get, isLoading, isSuccess, post } = useFetch();
   const { id } = useParams();
   const history = useHistory();
   const storeDispatch = useDispatch();
```

Comment:
Why not dispatch? There is no danger of confusion here

Reply:
@aditudorache this pull request is about the `component` not the `container` which was mainly  done by janjaap, please discuss with him ;)